spec/javascripts/support/jasmine.yml

```diff
@@ -1,34 +1,146 @@
+# Default: []
+#
+# EXAMPLE:
+#
+# src_files:
+#   - lib/source1.js
+#   - lib/source2.js
+#   - 'dist/**/*.js'
+#
 src_files:
- - "../../../spec/javascripts/helpers/jquery-1.12.4.js"
- - "../../../spec/javascripts/vendor/jasmine-jquery.js"
- - "application.js"
+  - "spec/javascripts/vendor/jquery-1.12.4.js"
+  - "spec/javascripts/vendor/jasmine-jquery.js"
+  - "assets/application.js"
 
-# path to parent directory of spec_files
-# relative path from Rails.root
-# defaults to spec/javascripts
-spec_dir: spec/javascripts
+# stylesheets
+#
+# Return an array of stylesheet filepaths relative to src_dir to include before jasmine specs.
+# Default: []
+#
+# EXAMPLE:
+#
+# stylesheets:
+#   - css/style.css
+#   - 'stylesheets/*.css'
+#
+stylesheets:
+  - "assets/application.css"
 
-# list of file expressions to include as helpers into spec runner
-# relative path from spec_dir
+# helpers
+#
+# Return an array of filepaths relative to spec_dir to include before jasmine specs.
+# Default: ["helpers/**/*.js"]
+#
+# EXAMPLE:
+#
+# helpers:
+#   - 'helpers/**/*.js'
+#
 helpers:
   - "helpers/**/*.js"
 
-# list of file expressions to include as specs into spec runner
-# relative path from spec_dir
+# spec_files
+#
+# Return an array of filepaths relative to spec_dir to include.
+# Default: ["**/*[sS]pec.js"]
+#
+# EXAMPLE:
+#
+# spec_files:
+#   - '**/*[sS]pec.js'
+#
 spec_files:
-  - "**/*[Ss]pec.js"
+  - "**/*[sS]pec.js"
+
+# src_dir
+#
+# Source directory path. Your src_files must be returned relative to this path. Will use root if left blank.
+# Default: project root
+#
+# EXAMPLE:
+#
+# src_dir: public
+#
+src_dir:
+
+# spec_dir
+#
+# Spec directory path. Your spec_files must be returned relative to this path.
+# Default: spec/javascripts
+#
+# EXAMPLE:
+#
+# spec_dir: spec/javascripts
+#
+spec_dir: "spec/javascripts"
+
+# spec_helper
+#
+# Ruby file that Jasmine server will require before starting.
+# Returned relative to your root path
+# Default spec/javascripts/support/jasmine_helper.rb
+#
+# EXAMPLE:
+#
+# spec_helper: spec/javascripts/support/jasmine_helper.rb
+#
+spec_helper: "spec/javascripts/support/jasmine_helper.rb"
+
+# boot_dir
+#
+# Boot directory path. Your boot_files must be returned relative to this path.
+# Default: Built in boot file
+#
+# EXAMPLE:
+#
+# boot_dir: spec/javascripts/support/boot
+#
+boot_dir:
+
+# boot_files
+#
+# Return an array of filepaths relative to boot_dir to include in order to boot Jasmine
+# Default: Built in boot file
+#
+# EXAMPLE
+#
+# boot_files:
+#   - '**/*.js'
+#
+boot_files:
+
+# rack_options
+#
+# Extra options to be passed to the rack server
+# by default, Port and AccessLog are passed.
+#
+# This is an advanced options, and left empty by default
+#
+# EXAMPLE
+#
+# rack_options:
+#   server: 'thin'
 
-# path to directory of temporary files
-# (spec runner and asset cache)
-# defaults to tmp/jasmine
-tmp_dir: "tmp/jasmine"
+# phantom_cli_options
```

Comment:
Given that we've removed the phantomjs dependency, perhaps we could also remove these commented lines?